cirq-aqt/cirq_aqt/aqt_sampler.py

```diff
@@ -40,16 +41,68 @@ class AQTSampler(cirq.Sampler):
     runs a single circuit or an entire sweep remotely
     """
 
-    def __init__(self, remote_host: str, access_token: str):
+    def __init__(self, workspace: str, resource: str, access_token: str, remote_host: str = "https://arnica.aqt.eu/api/v1/"):
```

Comment:
Yes, it was because of the default argument. We preferred the definition of workspace and resource than requiring the user to piece together a URL, but this does require breaking backwards-compatibility. We figured that by requiring two arguments now, it would be clear enough that the "old way" is not the right way any more.